src/chapters/setup-intro-data-types/setup.rst

```diff
@@ -1,2 +1,218 @@
+
+   }
+
+We'll discuss the syntax of this program soon, but you can likely trust your gut
+that this program has an expected output of "Hello, World". To test this hypothesis,
+open a terminal window and navigate to the parent directory of your new file. Run:
+
+.. sourcecode:: bash
+
+   java HelloWorld.java
+
+You should see your greeting printed! 
+
+Java version 11 introduced the capability to compile single-file Java programs
+without explicitly running a command to compile. If our ``Hello, World`` program
+were more complex and contained another file, we would need to first run ``javac HelloWorld.java``,
+to compile, followed by ``java HelloWorld.java``.
+
+
+Install IntelliJ
+-----------------
+
+IntelliJ is an **integrated development environment (IDE)**. An IDE is like a text
+editor on steroids. It's a program to not only write and edit your code, but an IDE
+can also contain many other features to enhance the coding experience. IntelliJ offers
+code completion hints, debugging, and even it's own compiler. We'll be using it throughout
+this course, so it's time to get familiar with some of the basics.
+
+Visit the `IntelliJ download site <https://www.jetbrains.com/idea/download/>`__.
+Select your operating system and the Community version. Follow the installation
+prompts to select your settings. When you reach the window asking for your UI theme,
+you can choose to *Skip Remaining and Set Defaults*. You will finish on an IntelliJ
+window listing the options to *Create New Project*, *Import Project*, *Open*, and 
+*Check out from Version Control*.
+
+.. figure:: figures/IntelliJWelcome.png
+   :scale: 80%
+   :alt: Welcome window for IntelliJ
+
+   IntelliJ welcome window
+
+You've installed IntelliJ and you're ready to start exploring its many features.
+
+.. _create-new-java-project:
+
+Your First Java Project
+------------------------
+
+Following the "Hello, World" trend, let's create a new IntelliJ project.
+
+#. Create a new folder to hold your Java practice files. Since you will be
+   creating lots of small projects as you move through this course, we
+   suggest that you also add sub-folders with names corresponding to the
+   related chapters and projects. Something like
+   ``java-practice/chapter-name/project-name``.
+#. Select the *Create New Project* option from the welcome
+   screen.
+
+   .. figure:: figures/IntelliJWelcome.png
+      :scale: 80%
+      :alt: Welcome window for IntelliJ
+
+      Create new project
+
+#. Clicking *New Project* opens a window with a series of project settings to
+   select. For this first setting, make sure your selected project SDK is the JDK
+   you have installed. This allows IntelliJ to compile our Java code in-app. 
+   Click *Next* in the lower right corner of the window to continue selecting settings.
+
+   .. figure:: figures/projectSDK.png
+      :alt: Select project SDK
+
+      Select project SDK
+
+#. In the second window, select *Create project from template*. This gives us
```

Comment:
Can we touch base about using a template vs. not.  The template adds a ``package`` to the project and some boilerplate code.  Is this necessary for such a simple program?  Should we explain ``package``? What are the benefits of using the template vs. having the students cut and paste the code for the method?